packages/sakuli-legacy/src/context/common/test-case/test-case.class.ts

```diff
@@ -42,7 +42,7 @@ export function createTestCaseClass(ctx: TestExecutionContext, project: Project,
             public _imagePaths: string[] = []
         ) {
             ctx.logger.info(`Start Testcase ${caseId}`);
-            ctx.startTestCase({id: caseId});
+            ctx.startTestCase({id: caseId, warningTime: warningTime, criticalTime: criticalTime});
```

Comment:
With ES2015 it's possible to use a shorthand notation to set object properties from variables with matching names, so we could shorten this call.
See [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#Property_definitions) for reference.